src/ert/job_queue/queue.py

```diff
@@ -110,8 +110,11 @@ def is_active(self) -> bool:
             for real in self._realizations
         )
 
-    def count_status(self, state: RealizationState) -> int:
-        return len([real for real in self._realizations if real.current_state == state])
+    def real_state(self, iens: int) -> RealizationState:
```

Comment:
Suggest using the full name, `realization_state`, if that's what it is. 'Real state' could be misinterpreted.

Reply:
Or potentially find another abbreviation, like `rlz`?

Reply:
`realization_state` sounds good enough!